app/models/post.rb

```diff
@@ -0,0 +1,11 @@
+class Post < ApplicationRecord
+  extend FriendlyId
+friendly_id :title, use: :slugged
+  validates :title, presence: true
+  validates :content, presence: true, length: { minimum: 2 }
+  
+
+  def should_generate_new_friendly_id
+    title_changed? || slug.blank?
```

Comment:
i did not find any slug field or new migration how come it is working?

have we added any migration for friendly id?